lib/core_types/bool.dart

```diff
@@ -27,4 +26,5 @@ ASTNode visitBoolMethods(ASTNode node, ASTNode left) {
       }
     default:
   }
+  return null;
```

Comment:
Do not return null. These changes are not also related to this pr.

Reply:
Throw exception instead